crates/functional/Cargo.toml

```diff
@@ -31,3 +31,5 @@ wasm-bindgen-test = "0.3.4"
 
 [features]
 wasm_test = []
+web_sys = ["yew/web_sys"]
+std_web = ["yew/std_web"]
```

Comment:
are these necessary?

Reply:
Hmm it depends on yew, which requires them to be specified. Is there a way to lookup the current features enabled for the dependencies elsewhere using some cfg magic?

Reply:
Hmm not that I know of.

> Hmm it depends on yew, which requires them to be specified

Yes, but any project that depends on yew will specify them. Functional yew will inherit them for free

Reply:
I was hoping that that would happen but it did not, maybe I'm missing something obvious? :thinking:

Reply:
Strange, do you have an example project? 

From this post: https://blog.rust-lang.org/inside-rust/2020/01/10/cargo-in-2020.html
>  Currently, features are unified for all uses of a dependency

Maybe something has changed in a recent version of Rust?